packages/app-project/src/components/Announcements/components/FinishedAnnouncement/FinishedAnnouncement.stories.js

```diff
@@ -22,7 +22,7 @@ function NextRouterStory(Story) {
 
 const mockStore = {
   project: {
-    baseUrl: '/projects/zookeeper/galaxy-zoo',
+    baseUrl: '/zookeeper/galaxy-zoo',
```

Comment:
These links have `/projects` prepended because the storybook doesn't get the app base path from `next.config.js`, so it has to be added by hand.